packages/forma-36-react-components/src/components/Dropdown/DropdownListItem/DropdownListItem.tsx

```diff
@@ -79,21 +79,15 @@ export const DropdownListItem = forwardRef<HTMLElement, DropdownListItemProps>(
 
         return (
           <Element
-            className={styles['DropdownListItem__button']}
-            data-test-id="cf-ui-dropdown-list-item-button"
-            onClick={(e: ReactMouseEvent) => {
-              if (!isDisabled && onClick) {
-                onClick(e);
-              }
-            }}
-            onMouseDown={(e: ReactMouseEvent) => {
-              if (!isDisabled && onMouseDown) {
-                onMouseDown(e);
-              }
-            }}
+            onClick={!isDisabled && onClick !== undefined ? onClick : undefined}
+            onMouseDown={
+              !isDisabled && onMouseDown !== undefined ? onMouseDown : undefined
+            }
             type="button"
             {...(href ? linkProps : buttonProps)}
-            {...otherProps}
+            {...props}
+            className={styles['DropdownListItem__button']}
+            data-test-id="cf-ui-dropdown-list-item-button"
```

Comment:
We probably override this default test-id in many places already, so I'd be careful changing this.

Reply:
You're right, I intended to restore [previous functionality](https://github.com/contentful/forma-36/blob/271861a49f288bc0b1f510d9c97f8ef97bb099d7/packages/forma-36-react-components/src/components/Dropdown/DropdownListItem/DropdownListItem.tsx) but it turns out that it actually let you overwrite the submenu toggle class.